apps/pipelines/migrations/0008_pipelinechatmessages_node_id.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 5.1 on 2024-10-17 14:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Nit: How about merging this and the previous migration?

Reply:
https://github.com/dimagi/open-chat-studio/pull/740/commits/d748ecf69ddf279c631019311c347ecf3cb1a04f